app/code/Magento/Sales/Test/Mftf/Test/StorefrontVerifyStoreViewCartShippingNameTest.xml

```diff
@@ -0,0 +1,63 @@
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontVerifyStoreViewCartShippingNameTest">
+        <annotations>
+            <features value="Sales"/>
+            <stories value="Cart shipping name"/>
+            <title value="Verify cart different shipping names for different store views"/>
+            <description value="Verify cart different shipping names for different store views"/>
+            <severity value="MAJOR"/>
+            <testCaseId value="MC-*"/>
+            <group value="sales"/>
+        </annotations>
+        <before>
+            <createData entity="Simple_US_Customer_CA" stepKey="createCustomer"/>
+            <createData entity="_defaultCategory" stepKey="createCategory"/>
+            <createData entity="_defaultProduct" stepKey="createSimpleProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminCreateStoreViewActionGroup" stepKey="createStoreView"/>
+            <magentoCLI command="config:set {{StorefrontEnableAddStoreCodeToUrls.path}} {{StorefrontEnableAddStoreCodeToUrls.value}}" stepKey="enableAddStoreCodeToUrls"/>
+        </before>
+        <after>
+            <deleteData createDataKey="createCustomer" stepKey="deleteCustomer"/>
+            <deleteData createDataKey="createSimpleProduct" stepKey="deleteSimpleProduct"/>
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+            <actionGroup ref="AdminDeleteStoreViewActionGroup" stepKey="deleteStoreView"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+            <magentoCLI command="config:set {{StorefrontDisableAddStoreCodeToUrls.path}} {{StorefrontDisableAddStoreCodeToUrls.value}}" stepKey="disableAddStoreCodeToUrls"/>
+        </after>
+        <actionGroup ref="AdminOpenShippingMethodsConfigPageActionGroup" stepKey="openShippingMethodConfigPage"/>
+        <actionGroup ref="AdminSwitchStoreViewActionGroup" stepKey="switchStoreViewAdmin"/>
+        <conditionalClick selector="{{AdminShippingMethodFlatRateSection.carriersFlatRateTab}}" dependentSelector="{{AdminShippingMethodFlatRateSection.carriersFlatRateTitle}}" visible="false" stepKey="expandTab"/>
+        <click selector="{{AdminShippingMethodFlatRateSection.carriersFlatRateTitle}}" stepKey="uncheckUseSystemValueForFlatRateTitle"/>
+        <fillField selector="{{AdminShippingMethodFlatRateSection.carriersFlatRateTitleField}}" userInput="Test carrier title name" stepKey="fillFieldTitle"/>
+        <click selector="{{AdminShippingMethodFlatRateSection.carriersSaveConfig}}" stepKey="saveShippingConfig"/>
+        <actionGroup ref="CliCacheFlushActionGroup" stepKey="cleanCache">
+            <argument name="tags" value=""/>
+        </actionGroup>
```

Comment:
Have you tested this without flushing cache? If that is absolutely required, please just clean config cache, not the whole cache.

Reply:
Has been removed.